Update Question template for new issues

designsimply opened the pull request:
Now that Gutenberg has shipped in WordPress 5.0, we should consider updating the Question template for new issues to gently help people move their Gutenberg help requests to the main support forum as a first stop. This is because the ideal workflow is to ask for help and complete any troubleshooting needed with support first, which can then be moved to GitHub as a bug report once sufficient details are uncovered. This will help reduce duplicate requests posted to GitHub, increase the quality of bug reports, and help developers working on the project focus on feature work and bug fixes for verified bug reports.

designsimply committed:
Update Question template for new issues

Now that Gutenberg has shipped in WordPress 5.0, we should consider updating the Question template for new issues to gently help people move their Gutenberg help requests to the main support forum as a first stop. This is because the ideal workflow is to ask for help and complete any troubleshooting needed with support first, which can then be moved to GitHub as a bug report once sufficient details are uncovered. This will help reduce duplicate requests, increase the quality of bug reports, and help developers working on the project focus on feature work and bug fixes for verified bug reports.

gziolo commented:
Looks good in my opinion. What should we do about all open questions? Should we also close them with a similar message?

youknowriad reviewed (approved):
LGTM 👍 Let's run this by support too.

designsimply commented:
Good question, thank you for asking it! We discussed this in the 2019-01-23 Core Editor Weekly Chat on WP Slack at https://wordpress.slack.com/archives/C02QB2JS7/p1548253285187700 and decided to scrub the currently 90 open `[Type] Help Request` issues manually then implement a change to the Questions template to add StackExchange as a second recommended option for getting technical support.

The reasons for not bulk closing in this case are:

- Some older help requests never received a reply (19 of 90).
- Some help requests may have been mislabeled if information in the initial report was very limited.
- 90 is manageable to work through manually and two volunteers stepped up to help coordinate that (clorith and myself).

designsimply committed:
Update Question template to add StackExchange

Add https://wordpress.stackexchange.com/ as a fallback suggested resource for where to get help with technical help requests.

Note: this was suggested in a Core Editor Weekly Chat on 2019-01-23 at https://wordpress.slack.com/archives/C02QB2JS7/p1548252431142200.

designsimply commented:
This issue has been added to the Agenda for January 24th Support Meeting at https://make.wordpress.org/support/2019/01/agenda-for-january-24th-support-meeting/#comment-172967.

youknowriad commented:
Thanks for the updates 👍

designsimply commented:
I met with the WordPress Core Support Team today to discuss changes to the Question template and we decided to:

- Add a line to ask people to search before posting!
- Move the handbook note up.
- Update the title to "Help Request".
- Update the description to "Please post help requests or ‘how to’ questions in support channels first".

Link to the discussion for reference: https://wordpress.slack.com/archives/C02RQC6RW/p1548349915905600

designsimply committed:
Update Question template to ask to search first

I met with the WordPress Core Support Team to discuss changes to the Question template and we decided to:

* Add a line to ask people to search before posting!
* Move the handbook note up.
* Update the title to "Help Request".
* Update the description to "Please post help requests or ‘how to’ questions in support channels first".

Link to the discussion for reference: https://wordpress.slack.com/archives/C02RQC6RW/p1548349915905600

designsimply committed:
Update Question template to move the handbook link to the end

Move the handbook note to the bottom, to be friendlier.

michelleweber commented:
Overall, it's fine! My one thought is that I might not start so many lines with "Please" -- it makes it sound a little repetitive and automated. I'd probably rephrase a bit to be more like, "For general help requests, please post in the support forum at X. Technical help requests have their own section of the support forum at Y."

designsimply committed:
Update Question template with editorial feedback

Mending the tone after an editorial review! 😍
